packages/zoe/src/contracts/multipoolAutoswap/getCurrentPrice.js

```diff
@@ -8,9 +8,106 @@ import '../../../exported';
  * @param {(brand: Brand) => boolean} isSecondary
```

Comment:
Nitpick, but in line 4, now four methods are returned rather than two.

Reply:
Thanks, that gave me an opportunity to describe the difference between the old and new.